src/cascadia/TerminalApp/ColorScheme.cpp

```diff
@@ -16,6 +16,26 @@ static const std::wstring NAME_KEY{ L"name" };
 static const std::wstring TABLE_KEY{ L"colors" };
 static const std::wstring FOREGROUND_KEY{ L"foreground" };
 static const std::wstring BACKGROUND_KEY{ L"background" };
+static const std::int16_t TABLE_SIZE = 16;
+static const std::wstring TABLE_COLORS[TABLE_SIZE] =

```

Comment:
If this were a std::array<std::wstring, TABLE_SIZE>, you could use a foreach style loop below which might be nicer than referring to `TABLE_SIZE` again for the loop size restriction below.

```
for (const auto& current : TABLE_COLORS) 
{
}
```

Reply:
Sorry, or even... `std::array<std::wstring, 16>` and not define the size separately at all.

Reply:
I'll give that a try, I was originally running into issues with `SIZE` against the array that caused me to switch to this.

To ensure the colours are inserted to the correct index of `_table`, I've been using `i` from the loop. If I switch to this `for` loop, is there an easy way to get the index of that item?

Reply:
Actually, I've updated with a local counter. Let me know if there's a better way to achieve it.

Reply:
Either way then. But I would do the `.at` thing I referenced in the other comment to ensure you get std::out_of_range_exception if they mismatch for some reason.